pom.xml

```diff
@@ -422,6 +423,12 @@
             <version>${version.jmockit}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.picketbox</groupId>
```

Comment:
Ah, I missed it was just test - if just test couldn't we just pre-generate some Strings to compare against?  I don't think we expect PicketBox to be a moving target we need to keep tracking.